spec/asyncapi.md

```diff
@@ -1234,6 +1235,55 @@ schemaFormat: 'application/vnd.apache.avro+yaml;version=1.9.0'
 contentType: application/json
 ```
 
+#### <a name="messageExampleObject"></a> Message Example Object
+
+Message Example Object represent example of a [Message Object](#messageObject) and MUST contain either **headers** and/or **payload** fields. 
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="messageExampleObjectHeaders"></a>headers | `Map[string, any]` | The value of this field MUST validate against the [Message Object](#messageObject) [headers](#messageObjectHeaders) field. 
```

Comment:
```suggestion
<a name="messageExampleObjectHeaders"></a>headers | `Map[string, any]` | The value of this field MUST validate against the [Message Object's headers](#messageObjectHeaders) field. 
```

Reply:
Sure, look better and `Message Object` string is still searchable by native browser capabilities (CTRL+F)

Reply:
Actually, the link takes you to the Message Object. Ok, to the `headers` field but still the Message Object 😄 That's why I did it.